docs/recipes/DisplayNameComponents.md

```diff
@@ -4,7 +4,7 @@ Sometimes, you need to have an explicit displayName for your components rather t
 
 ## Using a HoC
 
-One way to handle this problem is is by using a HoC. We recommend [recompose](https://github.com/acdlite/recompose).
+One way to handle this problem this is by using a HoC. We recommend [recompose](https://github.com/acdlite/recompose).
```

Comment:
The `this` shouldn't be there as well right? was just a duplicate `is`.

Reply:
Yep...fixed